modules/backend/formwidgets/datepicker/partials/_datepicker.htm

```diff
@@ -1,6 +1,6 @@
 <?php if ($this->previewMode): ?>
     <div class="form-control"><?= Backend::dateTime($value, [
-        'format' => $format,
+        'format' => $format . ' H:i',
```

Comment:
@AIC-BV wouldn't people just change the format if they want the time displayed, as opposed to forcing it onto the end of the string?

Reply:
Yes, I don't think we can accept this change without a further justification as to why it is required.

Reply:
Datetime in preview mode only shows date, not the time:
![image](https://user-images.githubusercontent.com/89913092/161091264-9d8a650b-3e9c-4e13-bb76-26720856352f.png)
To me, this feels like a bug because datetime is dateTIME not date, which is why I suggested adding H:i in preview mode.

Possible workaround:
```
created_at@preview:
    label: Created at
    type: datepicker
    mode: datetime
    format: d/m/Y H:i
    default: now

created_at:
    context: ['create', 'update']
    label: Created at
    type: datepicker
    mode: datetime
    format: d/m/Y
    default: now
```

Reply:
The correct behaviour would perhaps be to in the FormWidget class itself automatically set the format property if it hasn't been changed from the default when previewMode is true.

Reply:
So how does this continue?
What are the next steps? Do I have to look in to the FormWidget and come up with another solution?

Reply:
Instead of making the change here, add some code to DatePicker.php that checks if the format property hasn't been set, it's in previewMode, and the mode is datetime before it sets the format property itself to include the time.

Reply:
> if the format property hasn't been set

So in my case, where format is set to `d/m/Y` it won't work and the previewMode is still without time indicator 🤔

Reply:
> created_at@preview:
>     label: Created at
>     type: datepicker
>     mode: datetime
>     format: d/m/Y H:i
>     default: now
> 
> created_at:
>     context: ['create', 'update']
>     label: Created at
>     type: datepicker
>     mode: datetime
>     format: d/m/Y
>     default: now

If I understand you correctly, my only solution remains this?

Reply:
That looks good to me yep